test/e2e/apps/disruption.go

```diff
@@ -214,6 +196,34 @@ var _ = SIGDescribe("DisruptionController", func() {
 			}
 		})
 	}
+
+	ginkgo.It("should block an eviction until the PDB is updated to allow it", func() {
```

Comment:
is the blocking if PDB doesnt allow it , a new behavior ?

Reply:
It’s the standard behavior of the pdb to block an eviction unless it’s conditions have been satisfied.